hypershift-operator/controllers/nodepool/nodepool_controller.go

```diff
@@ -807,8 +819,12 @@ func (r *NodePoolReconciler) reconcile(ctx context.Context, hcluster *hyperv1.Ho
 	}
 
 	// Reconcile (Platform)MachineTemplate.
-	template, mutateTemplate, machineTemplateSpecJSON, err := machineTemplateBuilders(hcluster, nodePool, infraID, ami, powervsBootImage, kubevirtBootImage)
+	template, mutateTemplate, machineTemplateSpecJSON, err := machineTemplateBuilders(hcluster, nodePool, infraID, ami, powervsBootImage, kubevirtBootImage, cpoCapabilities.CreateDefaultAWSSecurityGroup)
 	if err != nil {
+		if _, isNotReady := err.(*NotReadyError); isNotReady {
```

Comment:
Note for ourselves: this is exactly the kind of change why we need to start gating CI on SLOs as this might impact time to Node available.

Reply:
Agree, this will definitely have an impact, which can be mitigated when you specify SG's on your NodePool to start with. You only have a penalty if not specifying a SG.